.github/workflows/ci.yml

```diff
@@ -61,3 +61,45 @@ jobs:
 
     steps:
       - run: echo "ci passed"
+
+  publish:
+    name: Publish to PyPI
+    runs-on: ubuntu-latest
+    needs: build-test-release
+    if: (github.event_name == 'workflow_dispatch') || github.event_name == 'push'
+
+    steps:
+      - name: Download assets from GitHub release
+        uses: robinraju/release-downloader@v1
+        with:
+          repository: ${{ github.repository }}
+          # download the latest release
+          latest: true
+          # don't download pre-releases
+          preRelease: false
```

Comment:
We decided to be able to upload a alpha version right? So this has to be changed back to true?

Reply:
i'd consider that out of scope of this PR. currently we are unable to upload to PyPI at all (not even `main`). Let's first fix that and then we can do follow-ups